### ISSUE TITLE
[6.0.3] travis-after-all checks for travis-ci.org, but job runs on Travis Pro

### ISSUE THREAD

**Issue by gr2m** [semantic-release/semantic-release#91]
I get this error

```
[travis-after-all] Failed to parse JSON from "https://api.travis-ci.org/builds/15794427".
```

**Comment by boennemann** [comment]
This is a limitation with travis-after-all, so I've opened an issue there: https://github.com/alrra/travis-after-all/issues/6

**Comment by boennemann** [comment]
Fixed via v6.3.2
